core/src/main/java/org/aerogear/mobile/core/http/OkHttpResponse.java

```diff
@@ -104,4 +104,16 @@ public String stringBody() {
             return "";
         }
     }
+
+    @Override
+    public boolean requestFailed() {
```

Comment:
@pb82 Ignore this comment - I just want to flag that this http wrapper is not properly implemented.
@secondsun @danielpassos  Flagging this as it's kinda poor api. Any reason why we did not used lambdas here?

Reply:
@wtrocki I added `requestFailed` and `getRequestError` because i couldn't see any other way know when a request failed. Trying to get the status of a failed request will result in a NPE. Happy to change this if there is a better solution.

Reply:
Peter that's not problem. I just wanted to mention that http api did not provide that and needs to be redesigned. This fix works for now and it's not the problem with your PR.